packages/turbo-types/scripts/generate-schema.ts

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env node
+
+import { writeFileSync } from "node:fs";
+import { join } from "node:path";
+import { createGenerator } from "ts-json-schema-generator";
+
+const __dirname = new URL(".", import.meta.url).pathname;
+const packageRoot = join(__dirname, "..", "src");
+
+/**
+ * post-process the schema recursively to:
+ * 1. replace any key named `defaultValue` with `default`
+ * 1. remove any backticks from the value
+ * 1. attempt to parsing the value as JSON (falling back, if not)
```

Comment:
```suggestion
 * 2. remove any backticks from the value
 * 3. attempt to parsing the value as JSON (falling back, if not)
```

Reply:
Just making sure you're aware: markdown rendering rules are in effect in JSDoc/TSDoc and this is [the standard](https://github.com/markdownlint/markdownlint/blob/main/docs/RULES.md#md029---ordered-list-item-prefix) way of doing things because it removes any room for error in misnumbering:

<img width="50%" src="https://github.com/user-attachments/assets/ad194864-6689-4491-804a-0a3f4302f843" />

<img width="25%" src="https://github.com/user-attachments/assets/19501754-219b-47ca-ad65-e929767c167f" />

with that in mind: do you want to change this?

Reply:
(talked to @chris-olszewski offline and he said to go ahead and merge! :smile:)